frontend/src/interfaces/IGame.ts

```diff
@@ -1,16 +1,30 @@
 import {GameId, Date, LanguageId, BggId} from '@/interfaces/types';
 
-export interface IGame {
-  bggId: BggId;
-  buyDate: Date;
-  extensions: Array<GameId>;
-  id: GameId;
-  isExtension: boolean;
-  languages: Array<LanguageId>;
-  name: string;
-  playersCountMax: number;
-  playersCountMin: number;
-  playTimeMax: number;
-  playTimeMin: number;
-  urlTT: string;
+export interface INonPersistedGame {
+  bggId?: BggId;
+  buyDate?: Date;
+  extensions?: Array<GameId>;
+  isExtension?: boolean;
+  languages?: Array<LanguageId>;
+  name?: string;
+  playersCountMax?: number;
+  playersCountMin?: number;
+  playTimeMax?: number;
+  playTimeMin?: number;
+  urlTT?: string;
+}
```

Comment:
Du coup, je me demande s'il faudrait pas extraire les interfaces hors du folder BE and FE, et ensuite basé notre implémentation sur ces interfaces ?

Reply:
On regardera une autre fois ptete.

Reply:
Yup could be a good idea, I added an issue for this. #21